cache/remotecache/registry/registry.go

```diff
@@ -165,6 +165,7 @@ func (dsl *withDistributionSourceLabel) SnapshotLabels(descs []ocispecs.Descript
 		labels = make(map[string]string)
 	}
 	maps.Copy(labels, estargz.SnapshotLabels(dsl.ref, descs, index))
+	labels["containerd.io/snapshot/image-ref"] = dsl.ref
```

Comment:
This is overlaybd-specific label, right? Can it be enabled only for overlaybd snapshotter?

Reply:
Or,  "containerd.io/snapshot/image-ref" should be clearly defined somewhere (e.g. by defining this in `util` pkg with comments)

Reply:
resolved.
we use "containerd.io/snapshot/cri.image-ref" to replace it